impl/proto/keytransparency_v1_service/keytransparency_v1_service.proto

```diff
@@ -70,3 +70,15 @@ service KeyTransparencyService {
     option (google.api.http) = { get: "/v1/domain/info" };
   }
 }
+
+// The KeyTransparencyAdminService API provides batch access to the directory of public keys. 
+service KeyTransparencyAdminService {
+  // BatchSetEntries uses an authorized_public key to perform a set request on multiple entries at once.  
```

Comment:
That's completely fine. There's no line limit but I thought we're keeping comments at 80 characters :)
If you think it looks good in your editor, I'm fine keeping it as is.